README.md

```diff
@@ -1039,6 +1041,52 @@ EXAMPLES
   $ affinidi project create-project --name "My project name"
 ```
 
+## `affinidi project edit-project-description`
```

Comment:
I see you merged these two commands into a single one, which is great. But you need to recreate the README